src/System.Net.WebSockets/src/System.Net.WebSockets.csproj

```diff
@@ -18,18 +19,31 @@
     <Compile Include="System\Net\WebSockets\WebSocketMessageType.cs" />
     <Compile Include="System\Net\WebSockets\WebSocketReceiveResult.cs" />
     <Compile Include="System\Net\WebSockets\WebSocketState.cs" />
+    <Compile Include="$(CommonPath)\System\Net\WebSockets\ManagedWebSocket.cs">
```

Comment:
The plan is now that we won't use this for UAP (or Windows) and so we may want to conditionally compile it for platforms where it's used to avoid unnecessary disk footprint.

@stephentoub what is the binary size increase due to adding the implementation? 

/cc @DavidGoll